### PR TITLE
Portrait Mode Fixes

### PR DESCRIPTION
**Pull request by kyle-emmerich**

### CODE REVIEW

File: CoreScriptsRoot/Modules/Settings/Utility.lua [hunk 1]
```diff
@@ -877,11 +874,8 @@ local function CreateDropDown(dropDownStringTable, startPosition, settingsHub)
 			if DropDownScrollingFrame.CanvasSize.Y.Offset < (DropDownFullscreenFrame.AbsoluteSize.Y - 10) then
 				DropDownSelectionFrame.Size = UDim2.new(0, dropDownWidth,
 														0,DropDownScrollingFrame.CanvasSize.Y.Offset + SCROLLING_FRAME_PIXEL_OFFSET)
-				DropDownSelectionFrame.Position = UDim2.new(0.5, -dropDownWidth / 2,
```
Comment (hockeychaos) [inline CoreScriptsRoot/Modules/Settings/Utility.lua]: why are we dropping positions here?

File: CoreScriptsRoot/Modules/Settings/Utility.lua [hunk 2]
```diff
@@ -1430,10 +1379,13 @@ local function ShowAlert(alertMessage, okButtonText, settingsHub, okPressedFunc,
 	local removeId = HttpService:GenerateGUID(false)
 
 	local destroyAlert = function(actionName, inputState)
-		if fixSettingsMenuVR and UserInputService.VREnabled and (inputState == Enum.UserInputState.Begin or inputState == Enum.UserInputState.Cancel) then
+		if VRService.VREnabled and (inputState == Enum.UserInputState.Begin or inputState == Enum.UserInputState.Cancel) then
+			return
+		end
+		if not AlertViewBacking then
```
Comment (hockeychaos) [inline CoreScriptsRoot/Modules/Settings/Utility.lua]: is this check new?

File: CoreScriptsRoot/Modules/Settings/Utility.lua (hunk 1 above)
Comment (kyle-emmerich) [inline CoreScriptsRoot/Modules/Settings/Utility.lua]: It's centered by AnchorPoint now!

File: CoreScriptsRoot/Modules/Settings/Utility.lua (hunk 2 above)
Comment (kyle-emmerich) [inline CoreScriptsRoot/Modules/Settings/Utility.lua]: Yeah, there was an error sometimes because of AlertViewBacking didn't exist.

**Pull request merged**